Sources/Get/Request.swift

```diff
@@ -2,9 +2,13 @@
 //
 // Copyright (c) 2021-2022 Alexander Grebenyuk (github.com/kean).
 
+#if compiler(>=5.6)
+@preconcurrency import Foundation
+#else
 import Foundation
+#endif
 
-public struct Request<Response> {
+public struct Request<Response>: Sendable {
```

Comment:
I think this should be `extension Response: Sendable where T: Sendable {}`. We should restrict the users to only using `Sendable` types are responses.